auth_utils.py

```diff
@@ -2,41 +2,47 @@
 import datetime
 from flask import request, jsonify
 from functools import wraps
+import logging
 import os
+from cryptography.fernet import Fernet
 
-SECRET_KEY = os.getenv('SECRET_KEY')
+SECRET_KEY = os.environ.get('SECRET_KEY')
+# Keys
+ferney_key = os.environ.get("FERNET_KEY")
+fernet = Fernet(ferney_key.encode())
 
+# logging config
+logging.basicConfig(level=logging.DEBUG, 
+                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
+                    handlers=[logging.FileHandler("app.log"), logging.StreamHandler()])
 
 def generate_token(user_id):
     payload = {
         'exp': datetime.datetime.now() + datetime.timedelta(hours=1),  # Token expiration time
         'iat': datetime.datetime.now(),  # Issued at time
         'sub': user_id  # Subject of the token (user ID)
     }
-    return jwt.encode(payload, SECRET_KEY, algorithm='HS256')
+    token = jwt.encode(payload, SECRET_KEY, algorithm='HS256')
+    logging.debug(f"Generated token for user{user_id}: {token}")
```

Comment:
Now look what you did - you generated a token and printed it in the log. Let's say I'm an engineer on your team, I can look at the logs and get your customer's token and access their data :) whoops. Rule of thumb - DO NOT PUT any sensitive information in your logs.